### PR TITLE
add homework6

### PR DESCRIPTION
**Pull request by DaniilKorinnyi**
Added homework6

### CODE REVIEW

File: homework6/task.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
+phone_book_list = [
+    ["Kate", "+096231453"],
+    ["Jack", "+097312414"],
+    ["Alice", "+063672317"]
+]
+phone_book = dict(phone_book_list)
+stats = len(phone_book)
```
Comment (klymenok) [inline homework6/task.py:7]: this variable should be updated every time when structure of the phonebook is changed. It is better to put it after 'stat' command or use `len(phone_book)` there

File: homework6/task.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        c = input()
+        phone_book[name] = c
+        print(phone_book)
+
+    if b == "add":
+        add()
+    elif b == "delete":
+        print("Write name: ")
+        name = input()
+        del phone_book[name]
```
Comment (klymenok) [inline homework6/task.py:25]: It is better to check if key exists before deleting it

File: homework6/task.py [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+while True:
+    print("What do you want to do: ")
+    b = input()
+
+    def add():
+        print("Write name: ")
+        name = input()
+        print("Write his/her number: ")
+        c = input()
+        phone_book[name] = c
```
Comment (klymenok) [inline homework6/task.py:17]: Records shouldn't be overridden according to the task. Please, make additional check here

**Review by klymenok — CHANGES_REQUESTED** [review]

**Review by DaniilKorinnyi — COMMENTED** [review]
Some correction in homework6

**Review by klymenok — APPROVED** [review]
Looks good!

**Comment by DaniilKorinnyi** [comment]
> Looks good!

Thanks!

**Pull request merged**